planner/core/planbuilder.go

```diff
@@ -762,6 +766,14 @@ func (b *PlanBuilder) buildAnalyze(as *ast.AnalyzeTableStmt) (Plan, error) {
 	return b.buildAnalyzeTable(as)
 }
 
+func buildShowNextRowID() *expression.Schema {
+	schema := expression.NewSchema(make([]*expression.Column, 0, 3)...)
+	schema.Append(buildColumn("", "Database_name", mysql.TypeVarchar, mysql.MaxDatabaseNameLength))
+	schema.Append(buildColumn("", "Table_name", mysql.TypeVarchar, mysql.MaxTableNameLength))
+	schema.Append(buildColumn("", "Next_global_row_ID", mysql.TypeLonglong, 4))
```

Comment:
How about adding another output field to show the column name. If we use the primary key as the table handle, we can directly print the primary key column, otherwise, just print "_tidb_rowid"